svc/pkg/game-user/ops/recommend/tests/integration.rs

```diff
@@ -1,6 +1,88 @@
 use chirp_worker::prelude::*;
+use proto::backend::{self, pkg::*};
+
+use game_user::create::Response as GameUser;
+
+async fn create_game_users(
+	ctx: &TestCtx,
+	count: usize,
+	namespace_id: Option<Uuid>,
+) -> (Uuid, Vec<GameUser>) {
+	let mut users = vec![];
+
+	let namespace_id = namespace_id.unwrap_or_else(Uuid::new_v4);
+
+	tracing::info!(?count, "Creating game users");
+
+	for i in 0..count {
+		let operation = op!([ctx] game_user_create {
+			namespace_id: Some(namespace_id.into()),
+			user_id: Some(Uuid::new_v4().into())
+		});
+
+		users.push(operation.await.unwrap());
+	}
+	tracing::info!(?count, "game users created");
+
+	(namespace_id, users)
+}
+
+#[worker_test]
+async fn test_return_ok(ctx: TestCtx) {
+	op!([ctx] game_user_recommend { count: 0 }).await.unwrap();
+}
+
+#[worker_test]
+async fn check_order(ctx: TestCtx) {
+	let generated_users = create_game_users(&ctx, 10, None).await.1;
+
+	let mut generated_user_ids = generated_users
+		.into_iter()
+		.map(|game_user| {
+			game_user
+				.game_user_id
+				.map(Uuid::from)
+				.ok_or("No game_user_id found in the game user.")
+		})
+		.collect::<Result<Vec<Uuid>, _>>()
+		.unwrap();
+
+	// So we don't have to worry about accidental mutability.
+	let generated_user_ids = {
+		let mut temp_gen_usr_ids = generated_user_ids;
+		temp_gen_usr_ids.reverse();
+		temp_gen_usr_ids
+	};
+
+	let returned_user_ids: Vec<_> = op!([ctx] game_user_recommend { count: 10 })
+		.await
+		.unwrap()
+		.game_user_ids;
+	let returned_user_ids: Vec<_> = returned_user_ids.into_iter().map(Uuid::from).collect();
```

Comment:
Nit: You could combine this variable definition with the above variable definition.

Assuming you made two definitions so that you can tell the compiler what type `.collect` should collect into, you can instead use the [turbofish syntax](https://techblog.tonsser.com/posts/what-is-rusts-turbofish): `.collect::<Vec<_>>()`

Nit 2: We tend to use `.as_uuid()` (`common::Uuid::as_uuid` in fully qualified form) instead of `Uuid::from` when converting between proto UUIDs and native UUIDs.